codegen/src/main/twirl/templates/JavaClient/Client.scala.txt

```diff
@@ -21,13 +21,17 @@ import static @{service.packageName}.@{service.name}.Serializers.*;
 
 import scala.concurrent.ExecutionContext;
 
+import akka.grpc.AkkaGrpcGenerated;
+
 @GenMethodImports(service)
 
+@@AkkaGrpcGenerated
 public abstract class @{service.name}Client extends @{service.name}ClientPowerApi implements @{service.name}, AkkaGrpcClient {
   public static final @{service.name}Client create(GrpcClientSettings settings, ClassicActorSystemProvider sys) {
     return new Default@{service.name}Client(settings, sys);
   }
 
+  @@AkkaGrpcGenerated
```

Comment:
I'm not sure this is still necessary since it's an inner class of a class that is already annotated with `AkkaGrpcGenerated`, but cant hurt ;)